Xamarin.Essentials/Types/DeviceCalendar.shared.cs

```diff
@@ -82,7 +84,9 @@ public class Event : IEvent
 
         public string Status { get; set; }
 
-        RecurrenceRule IEvent.RecurrancePattern { get; set; }
+        public IReadOnlyList<IAttendee> Attendees { get; set; }
+
+        public RecurrenceRule RecurrancePattern { get; set; }
```

Comment:
I think for the initial spec for reading calendar we should remove the recurrencerule and simply return a flag to indicate whether it's a recurring event. Recurrence rules aren't trivial and this will require more extensive testing/review to get it approved. We'll leave this PR to include it but will need to decide on whether we include recurrence in what we submit

Reply:
Yep, I'm planning to remove that functionality for the next PR, as I'm aiming to make the next PR one that we'll look to submit for an initial release. I'm aiming to only hold what I consider the most useful data for this release, which for event is looking like:
Id(Local and External?), Event Name, Location, Date/Time start, Date/Time end, Allday, Description

And potentially attendees, but not too sure yet, as if we get attendees would probably require their attendance status which may need other information as well.